src/main/java/com/stripe/net/RawRequestOptions.java

```diff
@@ -48,11 +48,6 @@ public static RawRequestOptionsBuilder builder() {
   public static final class RawRequestOptionsBuilder extends RequestOptions.RequestOptionsBuilder {
     private Map<String, String> additionalHeaders;
 
-    /** Constructs a raw request options builder with default values. */
```

Comment:
do we not need this constructor?

Reply:
It wasnt in the master branch, so I assumed we don't need it here either